### PR TITLE
Add version number of obtained JAR release when generating AMI

### PR DESCRIPTION
**Pull request by benjaminsherwood**

### CODE REVIEW

File: dks-host/dks-host-install.sh [hunk 1]
```diff
@@ -44,6 +44,8 @@ echo "JAR_DOWNLOAD_URL: $URL"
 curl "$URL" -L -o /tmp/dks.jar
 sudo useradd dks -m
 
+sudo echo ${VERSION} > /opt/dks/version
```
Comment (tea-short) [inline dks-host/dks-host-install.sh]: sudo and redirect won't work :)

File: hbase-to-mongo-export/htme-host-install.sh [hunk 2]
```diff
@@ -41,6 +41,8 @@ echo "JAR_DOWNLOAD_URL: $URL"
 curl "$URL" -L -o /tmp/htme.jar
 sudo useradd htme -m
 
+sudo echo ${VERSION} > /opt/htme/version
```
Comment (tea-short) [inline hbase-to-mongo-export/htme-host-install.sh]: same

File: snapshot-sender/snapshot-sender_install.sh [hunk 3]
```diff
@@ -44,6 +44,8 @@ sudo mkdir /var/log/snapshot-sender
 #sudo cp /tmp/ami-builder/snapshot-sender/snapshot-sender.sh              /opt/snapshot-sender/
 #sudo cp /tmp/ami-builder/snapshot-sender/snapshot-sender                 /etc/init.d/
 
+# sudo echo ${VERSION} > /opt/snapshot-sender/version
```
Comment (tea-short) [inline snapshot-sender/snapshot-sender_install.sh]: same

File: dks-host/dks-host-install.sh (hunk 1 above)
Comment (benjaminsherwood) [reply dks-host/dks-host-install.sh]: OK, so as you noted before. Does Packer actually require sudo for every step of its script?

Comment (tea-short) [reply dks-host/dks-host-install.sh]: I haven't tested it without sudo and previous code had sudo. If it runs under ec2-user or whatever then I'd expect it to need sudo. In other words, I don't know :)

Comment (benjaminsherwood) [reply dks-host/dks-host-install.sh]: OK, have pushed it to test :)

**Pull request merged**